proto/commonfate/access/v1alpha1/access.proto

```diff
@@ -115,14 +115,17 @@ message BatchEnsureResponse {
   string trace_id = 1;
 
 
-  repeated GrantState grants = 2;
+  // repeated GrantState grants = 2;
  
   // User-facing diagnostic warnings, for messages such as the user not having permissions
   // to request some of the entitlements they asked for.
   //
   // Rather than failing the entire operation the Access Handler returns warnings.
   repeated Diagnostic diagnostics = 6;
-  string request_id = 7;
+
+  repeated AccessRequest access_requests = 7;
```

Comment:
I think we're losing some information here by removing the `GrantState` entirely - in the dry run, `GrantState` contained information about the proposed changes which would be made.

We should introduce some sort of 'Changes' object onto this response so that our CLI can inform the user whether new Access Requests are going to be created, or whether there is already an existing Access Request for the entitlements they want.